website/docs/components/dropdown/partials/code/how-to-use.md

```diff
@@ -122,6 +93,78 @@ Pass any [icon](/icons/library) name to `@icon` to change the icon used in Toggl
 </Hds::Dropdown>
 ```
 
+### List placement
+
+By default, the list is positioned below the button, aligned to the right. To change the list position pass `bottom-left`, `top-left`, or `top-right` to `@listPosition` on the Dropdown component.
+
+```handlebars
+<Hds::Dropdown @listPosition="bottom-left" as |dd|>
+  <dd.ToggleButton @text="Text Toggle" />
+  <dd.Interactive @route="components" @text="Item One" />
+  <dd.Interactive @route="components" @text="Item Two" />
+  <dd.Interactive @route="components" @text="Item Three" />
+  <dd.Interactive @text="Item Four (closes on click)" {{on "click" dd.close}} />
+  <dd.Separator />
+  <dd.Interactive @route="components" @text="Delete" @color="critical" @icon="trash" />
+</Hds::Dropdown>
+```
+
+In contexts where the Dropdown needs to be _inline_, to inherit the alignment from a parent, you can use the `@isInline` argument (and set the `@listPosition` accordingly to your needs):
+
+```handlebars
+<div class="doc-dropdown-mock-text-align-right">
+  <Hds::Dropdown @isInline={{true}} @listPosition="bottom-right" as |dd|>
+    <dd.ToggleButton @text="Text Toggle" @color="secondary" />
+    <dd.Interactive @route="components" @text="Item One" />
+    <dd.Interactive @route="components" @text="Item Two" />
+    <dd.Interactive @route="components" @text="Item Three" />
+  </Hds::Dropdown>
+</div>
+```
+
+### List size
+
+If you need to more explicitly control the height or width of a list, any acceptable value (px, rem, em) can be declared:
+
+```handlebars
+<Hds::Dropdown @isInline={{true}} @height="250px" @width="250px" as |dd|>
+  <dd.ToggleButton @text="Text Toggle" />
+  <dd.Interactive @route="components" @text="Item One" />
+  <dd.Interactive @route="components" @text="Item Two" />
+  <dd.Interactive @route="components" @text="Item Three" />
+  <dd.Interactive @route="components" @text="Item Four" />
+  <dd.Interactive @route="components" @text="Item Five" />
+  <dd.Interactive @route="components" @text="Item Six" />
+  <dd.Interactive @route="components" @text="Item Seven" />
+</Hds::Dropdown>
+```
+
+### List footer
+
+It is possible that you may want to add a list footer for things like a set of buttons for a filter control:
+
+```handlebars
+<Hds::Dropdown @height="200px" as |dd|>
```

Comment:
Increasing the height as it's a little cramped and making sure there's a slight reveal of results below to reinforce that the list is scrollable. 

```suggestion
<Hds::Dropdown @height="284px" as |dd|>
```